src/true_peak.rs

```diff
@@ -164,11 +164,16 @@ impl TruePeak {
 
     pub fn check_true_peak<'a, T: Sample + 'a, S: crate::Samples<'a, T>>(
         &mut self,
-        src: &S,
+        src: S,
         peaks: &mut [f64],
     ) {
         self.interp.check_true_peak(src, peaks)
     }
+
+    pub fn seed<'a, T: Sample + 'a, S: crate::Samples<'a, T>>(&mut self, src: S) {
+        let mut true_peaks = vec![0.0; src.channels()];
```

Comment:
Sad heap allocation :) Maybe use a `SmallVec`here with a reasonable number of channels?

Reply:
Sure! Though, this is going to be done at most once per chunk of data analyzed. Probably not going to matter,